app/models/feedback_reporters/feedback_reporter.rb

```diff
@@ -24,31 +24,23 @@ def description
   end
 
   def send_report!
```

Comment:
Just checked our Slack archives and con confirm `ArchiveConfig.ABUSE_REPORTS_SITE` was also added as a temporary measure to fix https://otwarchive.atlassian.net/browse/AO3-4569 while the committees were using different sites.